easybuild/easyconfigs/c/cairo/cairo-1.14.10-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'cairo'
+version = '1.14.10'
+
+homepage = 'http://cairographics.org'
+description = """Cairo is a 2D graphics library with support for multiple output devices.
+ Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
+ PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+source_urls = ['http://cairographics.org/releases/']
+sources = [SOURCE_TAR_XZ]
+checksums = ['7e87878658f2c9951a14fc64114d4958c0e65ac47530b8ac3078b2ce41b66a09']
+
+builddependencies = [
+    ('binutils', '2.27'),
+]
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('freetype', '2.7.1', '-libpng-1.6.29'),
+    ('pixman', '0.34.0'),
+    ('expat', '2.2.0'),
+    ('GLib', '2.53.5'),
+    ('X11', '20170314'),
+]
+
+# disable symbol lookup, which requires -lbfd, to avoid link issues with (non-PIC) libiberty.a provided by GCC
+configopts = "--enable-symbol-lookup=no --enable-gobject=yes --enable-svg=yes --enable-tee=yes --enable-xlib-xcb "
+
+sanity_check_paths = {
+    'files': ['bin/cairo-trace', 'lib/cairo/libcairo-trace.so', 'lib/cairo/libcairo-trace.a',
+              'lib/libcairo.a', 'lib/libcairo-gobject.a', 'lib/libcairo-script-interpreter.a',
+              'lib/libcairo-gobject.so', 'lib/libcairo-script-interpreter.so', 'lib/libcairo.so'] +
```

Comment:
@boegel Shouldn't these be using `SHLIB_EXT` instead? Or would you say it doesn't matter because it's just the sanity check?